index.d.ts

```diff
@@ -1,3 +1,19 @@
+// TypeScript Version: 3.0
+
+export interface HeadersDefaults {
```

Comment:
Any particular reason to add this interface here at the beginning, rather than towards the end?

Reply:
Nope. I didn’t really think about placement.